test/new_rpe_test.jl

```diff
@@ -0,0 +1,335 @@
+
+using SpheriCart, StaticArrays, LinearAlgebra, RepLieGroups, WignerD, 
+      Combinatorics, Rotations
+using WignerD: wignerD
+using RepLieGroups.O3: Rot3DCoeffs, Rot3DCoeffs_real, Rot3DCoeffs_long, coco_dot
+using RepLieGroups: gram
+O3 = RepLieGroups.O3
+using Test
+
+
+function eval_cY(rbasis::SphericalHarmonics{LMAX}, 𝐫) where {LMAX}  
```

Comment:
Both P4ML and Sphericart should now have complex spherical harmonics available. I'm not sure this code is still needed?

Reply:
Ok, I will check the corresponding part and simplify this test soon).